### PR TITLE
[PyTorch] Fix hparams kaggle-cifar

### PR DESCRIPTION
**Pull request by AnirudhDagar**
*Description of changes:*
This PR addresses the issue of hyperparameters and training inconsistencies with pytorch kaggle-cifar10 section in #1775.

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1784/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1784/

File: chapter_computer-vision/kaggle-cifar10.md [hunk 1]
```diff
@@ -513,8 +513,8 @@ train(net, train_iter, valid_iter, num_epochs, lr, wd, devices, lr_period,
 
 ```{.python .input}
 #@tab pytorch
-devices, num_epochs, lr, wd = d2l.try_all_gpus(), 5, 0.1, 5e-4
-lr_period, lr_decay, net = 50, 0.1, get_net()
```
Comment (astonzhang) [inline chapter_computer-vision/kaggle-cifar10.md:517]: What did you observe when lr=0.02?

Comment (AnirudhDagar) [reply chapter_computer-vision/kaggle-cifar10.md:517]: With `lr=0.02`, train loss decreased a bit at the start and then was constant. While `train acc` and `valid acc` remained almost zero.
